src/swarm-engine/Swarm/Game/Step/Const.hs

```diff
@@ -16,26 +16,70 @@ import Control.Carrier.State.Lazy
 import Control.Effect.Error
 import Control.Effect.Lens
 import Control.Effect.Lift
-import Control.Lens as Lens hiding (Const, distrib, from, parts, use, uses, view, (%=), (+=), (.=), (<+=), (<>=))
-import Control.Monad (filterM, forM, forM_, guard, msum, unless, when)
+import Control.Lens as Lens hiding (
+  Const,
+  distrib,
+  from,
+  parts,
+  use,
+  uses,
+  view,
+  (%=),
+  (+=),
+  (.=),
+  (<+=),
+  (<>=),
+ )
```

Comment:
I assume splitting out all the import lists onto separate lines like this is something else your editor did automatically?  I do find it pretty annoying, it uses up a ton of vertical space for no good reason, especially when the list of imports is short.  If there's an easy way for you to revert these reformatted imports, that would be nice --- but ultimately it's not a big deal, don't worry about it if it would be a pain.

Reply:
I can revert no worries. I was hoping the auto format would fix it up.

Reply:
Feel free to add the `merge me` label once you're happy with it to kick off the merge bot.  Though I think the Windows build may be broken at the moment (https://github.com/chhackett/vty-windows/issues/19).